File: EstuSozlukAPI/EstuSozlukAPI/Models/Entry.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace EstuSozlukAPI.Models
{
    public class Entry
    {
        public Entry()
        {
            Answers = new List<Answer>();
        }
        public int Id { get; set; }

        public int UserId { get; set; }
        public string Title { get; set; }

        public string Description { get; set; }

        public DateTime Date { get; set; }

        public List<Answer> Answers { get; set; }
        public User User { get; set; }
    }
}

File: EstuSozluk-spa/src/app/models/entry.ts
import { Answer } from './answer';

export class Entry {
  id: number | undefined;
  title: string | undefined;
  description: string | undefined;
  category: string | undefined;
  userId: number | undefined;
  date: Date | undefined;
  answers: Answer[] | undefined;

  /*public constructor(id: number, title: string, userId: number, answers: Answer[], description: string, date: Date ) {
    this.id = id;
    this.title = title;
    this.userId = userId;
    this.answers = answers;
    this.description = description;
    this.date = date;
  }*/
}

